eth_account/messages.py

```diff
@@ -37,6 +48,8 @@ def defunct_hash_message(primitive=None, hexstr=None, text=None):
     :type primitive: bytes or int
     :param str hexstr: the message encoded as hex
     :param str text: the message as a series of unicode characters (a normal Py3 str)
+    :param bytes signature_version: a byte indicating which kind of prefix is to be added (EIP 191)
+    :param version_specific_data: the data which is needed for adding the prefix (EIP 191)
```

Comment:
I don't think the wording here is quite right. I don't think the data is needed for adding the prefix. I think it's just data. It is related to the prefix, but not needed for _adding_ the prefix. For example, in EIP 712 it's the domain separator. https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md#definition-of-domainseparator, and in EIP 191 it's an address like you have above.